app/components/SourceSelector.vue.ts

```diff
@@ -167,11 +174,26 @@ export default class SourceSelector extends Vue {
     const destNode = this.scene.getNode(position.node.data.id);
 
     if (position.placement === 'before') {
-      nodesToMove.placeBefore(destNode.id);
+      this.editorCommandsService.executeCommand(
+        'ReorderItemsCommand',
```

Comment:
Realizing now that to be correct this command should be renamed to `ReorderNodesCommand` because it operates on Nodes and not just Items.